src/Raven.Server/RavenServer.cs

```diff
@@ -2108,7 +2108,10 @@ private async Task DispatchTcpConnection(TcpConnectionHeaderMessage header, TcpC
                 //At this stage the error is not relevant.
 
                 if (header.LicensedFeatures != null)
-                    header.LicensedFeatures.DataCompression &= ServerStore.LicenseManager.LicenseStatus.HasTcpDataCompression;
+                {
+                    header.LicensedFeatures.DataCompression &= ServerStore.LicenseManager.LicenseStatus.HasTcpDataCompression && 
+                                                               Configuration.Databases.DisableTcpCompression == false;
```

Comment:
this one is server-wide, do we even can override anything per database here?

Reply:
is this the place that negotiates also subscriptions? If yes, it looks like we cannot override it per database. If this is true, then we need to move the option from DatabaseConfiguration to a server one or tcp one? Tcp.Compression.Disable maybe?

Reply:
I would like it better if this was a server wide option as well.
Can't imagine the scenario where this is required on a per database level, and don't want to support that.

Reply:
changed to ServerWideOnly